bzip2.rs

```diff
@@ -1571,7 +1553,7 @@ unsafe fn uncompress(name: Option<String>) {
     if verbosity >= 1 {
         eprint!("  {}: ", in_name.display(),);
         pad(inName.as_mut_ptr());
-        fflush(stderr);
+        let _ = std::io::stderr().flush();
```

Comment:
Flushing for `std::io::stderr()` isn't really necessary. In C it may be necessary if libc buffers stderr, but libstd only line buffers stdout. Each individual write to stderr is immediately written to the underlying fd.

Reply:
I'll just remove the flushing then